apps/mobile/app/dashboard/lists/[slug]/edit.tsx

```diff
@@ -0,0 +1,113 @@
+import { useState } from "react";
+import { View } from "react-native";
+import { useLocalSearchParams } from "expo-router";
+import { dismiss } from "expo-router/build/global-state/routing";
+import { Button } from "@/components/ui/Button";
+import CustomSafeAreaView from "@/components/ui/CustomSafeAreaView";
+import { Input } from "@/components/ui/Input";
+import { Text } from "@/components/ui/Text";
+import { useToast } from "@/components/ui/Toast";
+import { api } from "@/lib/trpc";
+
+import { useEditBookmarkList } from "@karakeep/shared-react/hooks/lists";
+
+// TODO Duplicated from new -- import from somewhere
+type ListType = "manual" | "smart";
+
+const EditListPage = () => {
+  const [text, setText] = useState("");
+  const [listType, setListType] = useState<ListType>("manual");
```

Comment:
List types are typically not editable. So this probably shouldn't be a state but instead read directly form `list.type`